cluster/sync.sh

```diff
@@ -62,6 +62,9 @@ function deploy() {
     # Build new handler image from local sources and push it to the kubevirtci cluster
     IMAGE_REGISTRY=${registry} make push-handler
 
+    # Build new operator image from local sources and push it to the kubevirtci cluster
+    IMAGE_REGISTRY=${registry} make push-operator
```

Comment:
we can do `IMAGE_REGISTRY=${registry} make push-operator push-handler` or even add a target that do both 
`IMAGE_REGISTRY=${registry} make push` at the end we are going to need it too to publish both a quay and the postsubmit prow job.